File: python/koans_4_bonus.py
class Koans4Bonus :
    def goToTest(self) :
        # by using ⌘⇧T / Ctrl+Shift+T (on the class name)
        raise Exception('remove me and go back to test')
        return 'go to test'

    def inside(self) :
        # Inline following method with ⌥⌘N / Ctrl+Alt+N
        # and change returned string for inside requirement
        return self.__inlinedMe()

    def outside(self) :
        return self.__inlinedMe()

    def __inlinedMe(self) :
        return 'outside'

File: java/src/main/java/Koans4Bonus.java
public class Koans4Bonus {
    public String goToTest() {
        // by using ⌘⇧T / Ctrl+Shift+T
        return "remove me, uncomment the following line and go back to test";
        // return "go to test";
    }

    public String inside() {
        // Inline following method with ⌥⌘N / Ctrl+Alt+N
        // and change returned string for inside requirement
        return inLineMe();
    }
    public String outside() {
        return inLineMe();
    }

    private String inLineMe() {
        return "outside";
    }
}
